module-contrib/scheduler.spec

```diff
@@ -61,30 +60,30 @@ rm -f %{buildroot}%{_localstatedir}/plugsched/%{KVER}-%{KREL}.%{_arch}/plugsched
 sync
 
 if [ "$(uname -r)" != "%{KVER}-%{KREL}.%{_arch}" ]; then
-	echo "INFO: scheduler dose not match kernel, skip load module..."
+	echo "INFO: scheduler dose not match kernel, skip starting service and loading module..."
```

Comment:
dose --> does

scheduler dose not match kernel  ---> scheduler does not match current kernel version, skip starting service ...